keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -320,95 +382,50 @@ needed.
 
 ### Rollout, Upgrade and Rollback Planning
 
-<!--
-This section must be completed when targeting beta to a release.
--->
-
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-<!--
-Try to be as paranoid as possible - e.g., what if some components will restart
-mid-rollout?
-
-Be sure to consider highly-available clusters, where, for example,
-feature flags will be enabled on some API servers and not others during the
-rollout. Similarly, consider large clusters and how enablement/disablement
-will rollout across nodes.
--->
+Not a workload feature, so it cannot impact running workload.
+API servers with feature enabled will experience different latency and throughput of LIST requests.
+If the apiserver watch cache is lagging it might cause a LIST requests to fail.
 
 ###### What specific metrics should inform a rollback?
 
-<!--
-What signals should users be paying attention to when the feature is young
-that might indicate a serious problem?
--->
+Users should check for increase if their apiserver latency and
+the `apiserver_watch_cache_read_wait` metric for direct impact of this feature to it.
+
+We expect 99th percentile of `apiserver_watch_cache_read_wait` to be below 200ms.
```

Comment:
Just to clarify: the 200ms was taken as 2 periods of progress notify? If so, can you please add that to the text?

Reply:
Done